native/android/gradle.properties

```diff
@@ -25,10 +25,7 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.176.0
-
-# Overwrite minSdkVersion of react-native-static-server
-RNStaticServer_minSdkVersion=21
```

Comment:
Why did you remove this?

Reply:
we don't use react-native-static-server anymore and the new library doesn't seem to use this constant either: https://github.com/search?q=repo%3Abirdofpreyru%2Freact-native-static-server%20RNStaticServer_minSdkVersion&type=code

Reply:
Ahh good find!